### PR TITLE
Delete Address Class from Taskify

### LINKED ISSUE #52: Delete Address class

### PR DESCRIPTION
**Pull request by hengyongming**
#54

### CODE REVIEW

**Review by noelmathewisaac — APPROVED** [review]
LGTM

File: src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java [hunk 1]
```diff
@@ -83,57 +74,50 @@ public void parse_compulsoryFieldMissing_failure() {
         String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE);
 
         // missing name prefix
-        assertParseFailure(parser, VALID_NAME_BOB + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB
+        assertParseFailure(parser, VALID_NAME_BOB + DESCRIPTION_DESC_BOB
                         + DATE_DESC_BOB, expectedMessage);
 
         // missing phone prefix
```
Comment (CSmortal) [inline src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java]: change the comment to "// missing description prefix "

File: src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java [hunk 2]
```diff
@@ -83,57 +74,50 @@ public void parse_compulsoryFieldMissing_failure() {
...
     }
 
     @Test
     public void parse_invalidValue_failure() {
         // invalid name
-        assertParseFailure(parser, INVALID_NAME_DESC + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB
+        assertParseFailure(parser, INVALID_NAME_DESC + DESCRIPTION_DESC_BOB
                         + DATE_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Name.MESSAGE_CONSTRAINTS);
 
         // invalid phone
```
Comment (CSmortal) [inline src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java]: change comment to "invalid description"

File: src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java [hunk 3]
```diff
@@ -83,57 +74,50 @@ public void parse_compulsoryFieldMissing_failure() {
...
-        assertParseFailure(parser, NAME_DESC_BOB + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB
+        assertParseFailure(parser, NAME_DESC_BOB + DESCRIPTION_DESC_BOB
                         + INVALID_DATE_DESC + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Date.MESSAGE_CONSTRAINTS);
 
         // invalid tag
-        assertParseFailure(parser, NAME_DESC_BOB + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB
+        assertParseFailure(parser, NAME_DESC_BOB + DESCRIPTION_DESC_BOB
                         + DATE_DESC_BOB + INVALID_TAG_DESC + VALID_TAG_FRIEND, Tag.MESSAGE_CONSTRAINTS);
 
         // two invalid values, only first invalid value reported
```
Comment (CSmortal) [inline src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java:114]: Following the comment, the assertParseFailure() in line 115 should have 2 invalid values. Can change either the description or date to an invalid value

File: src/test/java/seedu/taskify/logic/parser/EditCommandParserTest.java [hunk 4]
```diff
@@ -105,18 +98,18 @@ public void parse_invalidValue_failure() {
         assertParseFailure(parser, "1" + TAG_EMPTY + TAG_DESC_FRIEND + TAG_DESC_HUSBAND, Tag.MESSAGE_CONSTRAINTS);
 
         // multiple invalid values, but only the first invalid value is captured
-        assertParseFailure(parser, "1" + INVALID_NAME_DESC + INVALID_ADDRESS_DESC
+        assertParseFailure(parser, "1" + INVALID_NAME_DESC
                 + VALID_DESCRIPTION_AMY, Name.MESSAGE_CONSTRAINTS);
     }
```
Comment (CSmortal) [inline src/test/java/seedu/taskify/logic/parser/EditCommandParserTest.java]: similar to a previous comment, should change the description to an invalid one

**Review by CSmortal — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/taskify/logic/parser/EditCommandParserTest.java (hunk 4 above)
Comment (hengyongming) [reply src/test/java/seedu/taskify/logic/parser/EditCommandParserTest.java:103]: done

File: src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java (hunk 3 above)
Comment (hengyongming) [reply src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java:114]: done

File: src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java (hunk 2 above)
Comment (hengyongming) [reply src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java]: done

File: src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java (hunk 1 above)
Comment (hengyongming) [reply src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java]: done

**Comment by CSmortal** [comment]
LGTM!

**Review by CSmortal — APPROVED** [review]
LGTM!

**Comment by noelmathewisaac** [comment]
Closes #52

**Pull request merged**